src/index.js

```diff
@@ -68,11 +68,13 @@
     }
 
     //
+    this.alignOuput = opts.alignOuput === true ? true : false
```

Comment:
Lint is failing here
Guess `this.alignOutput = Boolean(opts.alignOutput)` will do the trick